### ISSUE TITLE
Braille panel doesn't obey UI text settings in Preferences > Appearance

### ISSUE THREAD

**Issue by shoogle** [musescore/MuseScore#21576]
### Issue type

UI bug (incorrect info or interface appearance)

### Bug description

Braille characters in the braille panel are a fixed size and colour (black). They don't change when the UI font size or colour is changed in **Preferences > Appearance**.

### Steps to reproduce

1. Go to **Preferences > Braille > Show braille panel**.
2. Go to **Preferences > Appearance**.
    1. Change **theme** from Light to Dark.
    2. Change **body text size**.

Observe that braille text in the braille panel is not affected by these settings.

### Screenshots/Screen recordings

Light theme, 12pt | Dark theme, 18pt
:---:|:---:
![image](https://github.com/musescore/MuseScore/assets/11011881/1ab85706-5118-48f0-bc45-fb978fc8138f) | ![image](https://github.com/musescore/MuseScore/assets/11011881/b9563100-1733-4cae-b21b-c04009e2e9eb)

### MuseScore Version

Latest nightly on master branch (1fe047c)

### Regression

No.

### Operating system

Windows 11

### Additional context

The braille font doesn't need to match the setting for the UI text font. In fact, it's probably best if a fixed font is used because some fonts don't contain glyphs for braille characters.

**Comment by re1san** [comment]
Hey, I would like to work on it :D

**Comment by nasehim7** [comment]
Hi @cbjeukendrup @re1san if there has not been substantial work done on this front, I am interested in working on this

**Comment by re1san** [comment]
@nasehim7 Working on it :)
